### ISSUE TITLE
DirFileSystem.ls() has `detail=True` as default

### ISSUE THREAD

**Issue by benlindsay** [fsspec/filesystem_spec#899]
In the DirFileSystem implementation, the `detail` parameter is set to `True` by default (see [here](https://github.com/fsspec/filesystem_spec/blob/master/fsspec/implementations/dirfs.py#L209) and [here](https://github.com/fsspec/filesystem_spec/blob/master/fsspec/implementations/dirfs.py#L200)) whereas for other implementations like LocalFileSystem, it is set to `False` (see [here](https://github.com/fsspec/filesystem_spec/blob/master/fsspec/implementations/local.py#L53)). This makes it so that `ls()` will give a list of dictionaries instead of a list of strings. Can we set that parameter to `False` by default so it has the expected behavior?

**Comment by martindurant** [comment]
Yes, we would like to aim for consistency across the implementations. I suspect this particular argument may be inconsistent in more places than the one you noticed.

**Comment by benlindsay** [comment]
Oh interesting, I see that the AbstractFileSystem sets `detail=True` by default, so I guess the cleanup should go in the other direction...maybe LocalFileSystem and others should change theirs to `detail=True`

**Comment by benlindsay** [comment]
Not super related, but is there a plan to add another release soon? I'd love to use DirFileSystem in an internal app at work but it's hard without a release. Thanks!

**Comment by martindurant** [comment]
In theory, we can release whenever we have accumulated enough new stuff or something critical comes up. It doesn't cost me too. much effort. We've been doing one or two releases a month.

**Comment by benlindsay** [comment]
@martindurant Any updates on when a new release might come out? Still really looking forward to using DirFileSystem in a project. Thanks in advance!!

**Comment by martindurant** [comment]
On PyPI now

**Comment by benlindsay** [comment]
woohoo, thank you!!
